### PR TITLE
Update README.md with JuliaAcademy reference

### PR DESCRIPTION
**Pull request by logankilpatrick**
Add a reference to JuliaAcademy

### CODE REVIEW

**Comment by bkamins** [comment]
Let us just wait for @nalimilan to have a look.

**Comment by nalimilan** [comment]
Why not, but maybe we should add a few more references (notably @bkamins's tutorials) in the manual too?

**Comment by bkamins** [comment]
I am OK with adding it as I plan to support it in long term. @logankilpatrick - if you are also OK with adding it then this is the link to the tutorial: https://github.com/bkamins/Julia-DataFrames-Tutorial/. Thank you!

**Comment by bkamins** [comment]
@logankilpatrick - how do you see this proposal?

**Comment by logankilpatrick** [comment]
What do you need on my end?

**Comment by bkamins** [comment]
It is your PR, so I assumed that you would decide what you prefer and propose a final wording (either leave things as is or incorporate the other link)

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Tools for working with tabular data in Julia.
 **Maintenance**: DataFrames is maintained collectively by the [JuliaData collaborators](https://github.com/orgs/JuliaData/people).
 Responsiveness to pull requests and issues can vary, depending on the availability of key collaborators.
 
+**Learning**: New to DataFrames.jl? Check out our [free Julia Academy course](https://juliaacademy.com/p/introduction-to-dataframes-jl) which will walk you through how to use DataFrames.jl.
```
Comment (bkamins) [inline README.md:18]:
```suggestion
**Learning**: New to DataFrames.jl? Check out our
[free Julia Academy course](https://juliaacademy.com/p/introduction-to-dataframes-jl)
which will walk you through how to use DataFrames.jl.
You can also check out
[the DataFrames.jl tutorial](https://github.com/bkamins/Julia-DataFrames-Tutorial)
that is available on GitHub.
```

**Comment by bkamins** [comment]
@nalimilan - is wording ok after the update I have just made?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,13 @@ Tools for working with tabular data in Julia.
 **Maintenance**: DataFrames is maintained collectively by the [JuliaData collaborators](https://github.com/orgs/JuliaData/people).
 Responsiveness to pull requests and issues can vary, depending on the availability of key collaborators.
 
+**Learning**: New to DataFrames.jl? Check out our
+[free Julia Academy course](https://juliaacademy.com/p/introduction-to-dataframes-jl)
+which will walk you through how to use DataFrames.jl.
+You can also check out
+[the DataFrames.jl tutorial](https://github.com/bkamins/Julia-DataFrames-Tutorial)
```
Comment (nalimilan) [inline README.md:22]: Don't be too modest. ;-)
```suggestion
[Bogumił Kamiński's DataFrames.jl tutorial](https://github.com/bkamins/Julia-DataFrames-Tutorial)
```

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**